src/__tests__/.next/_app.js

```diff
@@ -0,0 +1 @@
+// _app.js
```

Comment:
do we need that?

Reply:
Yes, it's a mocked next-app that we use to test the implementation